docs/hub/spaces-sdks-docker-argilla.md

```diff
@@ -1,199 +1,4 @@
 # Argilla on Spaces
+Argilla is a free and open source tool to build and iterate on data for AI. It can be deployed on the Hub with a few clicks and Hugging Face OAuth enabled, perfect for running community annotation initiatives!
```

Comment:
```suggestion
Argilla is a free and open source tool to build and iterate on data for AI. It can be deployed on the Hub with a few clicks and Hugging Face OAuth enabled. This enables other HF users to join your Argilla server to annotate datasets, perfect for running community annotation initiatives!

With Argilla you can:
- Configure datasets for collecting human feedback with a growing number questions (Label, NER, Ranking, Rating, free text, etc.)
- Use model outputs/predictions to evaluate them or speeding up the annotation process.
- UI users can explore, find, and label the most interesting/critical subsets using Argilla's search and semantic similarity features.
- Pull and push datasets from the Hugging Face Hub for versioning and model training.

```